test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -199,3 +255,38 @@ func runCpaSampleCode(ctx context.Context, f *framework.Framework, runTests int,
 	err = e2epod.WaitForPodSuccessInNamespaceTimeout(ctx, f.ClientSet, pod.ObjectMeta.Name, f.Namespace.Name, 300*time.Second)
 	gomega.Expect(err).To(gomega.BeNil(), utils.GetPodLogs(ctx, f, pod.ObjectMeta.Name, pod.Spec.Containers[0].Name))
 }
+
+func injectError(ctx context.Context, f *framework.Framework, resourceName v1.ResourceName) {
+	nodeName, _ := utils.FindNodeAndResourceCapacity(f, ctx, resourceName.String())
+	yes := true
+
+	job := e2ejob.NewTestJobOnNode("success", "qat-inject-error", v1.RestartPolicyNever, 1, 1, nil, 0, nodeName)
+	job.Spec.Template.Spec.Containers[0].Command = []string{
+		"/bin/sh",
+	}
+	job.Spec.Template.Spec.Containers[0].Args = []string{
+		"-c",
+		"find /sys/kernel/debug/qat_*/heartbeat/ -name inject_error -exec sh -c 'echo 1 > {}' \\;",
```

Comment:
if you cannot get this to work on multiple lines, just make them all `Command`. As it is now does not meet what I had in mind.

Reply:
Done!